plugin/main/src/kotlinx/benchmark/gradle/JsEngineExecTasks.kt

```diff
@@ -28,7 +28,7 @@ fun Project.createJsEngineBenchmarkExecTask(
     }
 
     if (compilationTarget is KotlinWasmSubTargetContainerDsl) {
-        check(compilation is KotlinJsIrCompilation) { "Legacy Kotlin/JS is does not supported by D8 engine" }
+        check(compilation is KotlinJsIrCompilation) { "Legacy Kotlin/JS is not supported by D8 engine" }
```

Comment:
Is it possible to change the `compilation` argument type to `KotlinJsIrCompilation`?

Reply:
The type was not modified in this case but when it is an IR compilation then the type is KotlinJsIrCompilation.